src/styles/_utils.scss

```diff
@@ -0,0 +1,13 @@
+$colors:
+  ".nav__link" #00acdc,
+  ".catalog__card" #34568b;
+
+@each $class, $color in $colors {
+  #{$class}:hover {
+    color: $color;
+  }
+}
+
+%duration {
+  transition: 300ms;
```

Comment:
transition is a shorthand property, use `transition-duration` if you only style the duration. And specify what properties you want to be affected by this rule.